test/test_framework/csm_test.py

```diff
@@ -120,6 +122,7 @@ def tmain(argp, argv):
 
 if __name__ == '__main__':
     sys.path.append(os.path.join(os.path.dirname(pathlib.Path(__file__)), '..', '..', '..'))
+    sys.path.append(os.path.join(os.path.dirname(pathlib.Path(os.path.realpath(sys.argv[0]))), '..', '..'))
```

Comment:
One more use of `sys.argv[0]` to define contents of the path list.  In this case it does not seem so bad, though, since it is only used by our test suite.

Reply:
Comment addressed.